code/modules/clothing/suits/hoodies.dm

```diff
@@ -1,51 +1,23 @@
 //Hoods for winter coats and chaplain hoodie etc
 
 /obj/item/clothing/suit/storage/hooded
-	var/obj/item/clothing/head/winterhood/hood
-	var/hoodtype = null
-	var/suittoggled = FALSE
 	var/opened = FALSE
 	sprite_sheets = list(
 		BODYTYPE_VAURCA_BULWARK = 'icons/mob/species/bulwark/hoodie.dmi'
 	)
+	var/hoodtype = /obj/item/clothing/head/winterhood
```

Comment:
the hoodtype is fine because the parent suit still needs to spawn it, so this can stay